client/tests/integration/transfer_asset.rs

```diff
@@ -55,6 +64,10 @@ fn simulate_insufficient_funds() {
         10_800,
     )
 }
+#[test]
+fn simulate_transfer_store_asset() {
+    simulate_transfer_store(Transfer::asset_store)
+}
```

Comment:
Looks like you can't use generic `simulate_transfer()` test here because of specific logic.

So let's just move `simulate_transfer_store()` code here

Reply:
done